Makefile

```diff
@@ -308,6 +309,7 @@ $(BUILD_DIR)/image/%/$(DUMMY): Makefile $(BUILD_DIR)/image/%/payload $(BUILD_DIR
 	$(DBUILD) -t $(DOCKER_NS)/fabric-$(TARGET) $(@D)
 	docker tag $(DOCKER_NS)/fabric-$(TARGET) $(DOCKER_NS)/fabric-$(TARGET):$(DOCKER_TAG)
 	docker tag $(DOCKER_NS)/fabric-$(TARGET) $(DOCKER_NS)/fabric-$(TARGET):$(ARCH)-latest
+	docker tag $(DOCKER_NS)/fabric-$(TARGET) $(DOCKER_NS)/fabric-$(TARGET)
```

Comment:
Why is this extra tag needed?

Reply:
The `dockerutil` actually uses https://github.com/hyperledger/fabric/blob/master/common/metadata/metadata.go#L10 to derive the tag. So during testing it uses the `latest` tag. When the binaries are actually built metadata.go is linked using `ldflags` and the value is actually propogated from the Makefile. But this doesn't occur in testing. 

We actually had to do the same thing in the master branches and release-2.x branches. But Matt actually gave us a workaround that removed the dependency on all docker images for unit tests. Unfortunately it's not as straight-forward for release-1.4 so simply adding an additional tag to the make target seems simplest as it doesn't change any of the underlying code or functionality.

So what used to happen (and we didn't realize) was we were actually pulling ccenv/baseos from Dockerhub during testing instead of using the local build.